Userland/Libraries/LibGfx/Bitmap.cpp

```diff
@@ -459,6 +459,17 @@ ErrorOr<NonnullRefPtr<Gfx::Bitmap>> Bitmap::cropped(Gfx::IntRect crop) const
     return new_bitmap;
 }
 
+ErrorOr<NonnullRefPtr<Gfx::Bitmap>> Bitmap::inverted() const
+{
+    auto new_bitmap = TRY(clone());
```

Comment:
I think `clone` is a bit overkill here,
You are not using the stored data and are just overriding it directly, so
we can save a memcpy by just doign a
```suggestion
    auto new_bitmap = TRY(Bitmap::try_create(format(), size(), scale()));
```

Reply:
Ah okay. Just thinking for this PR I think any allocations at all could be avoided if this method was changed to `invert()` to invert the pixels in-place, then just invert them again after painting.